crates/next-core/src/emit.rs

```diff
@@ -41,53 +41,51 @@ pub async fn emit_assets(
     node_root: Vc<FileSystemPath>,
     client_relative_path: Vc<FileSystemPath>,
     client_output_path: Vc<FileSystemPath>,
-) -> Result<Vc<Completion>> {
-    Ok(Vc::<Completions>::cell(
-        assets
-            .await?
-            .iter()
-            .copied()
-            .map(|asset| async move {
-                let asset = asset.resolve().await?;
-                let path = asset.ident().path();
-                let span =
-                    tracing::info_span!("emit asset", name = path.to_string().await?.as_str());
-                async move {
-                    let path = path.await?;
-                    Ok(if path.is_inside_ref(&*node_root.await?) {
-                        Some(emit(asset))
-                    } else if path.is_inside_ref(&*client_relative_path.await?) {
-                        // Client assets are emitted to the client output path, which is prefixed
-                        // with _next. We need to rebase them to remove that
-                        // prefix.
-                        Some(emit_rebase(asset, client_relative_path, client_output_path))
-                    } else {
-                        None
-                    })
-                }
-                .instrument(span)
-                .await
-            })
-            .try_flat_join()
-            .await?,
-    )
-    .completed())
+) -> Result<()> {
+    let _: Vec<Vc<()>> = assets
+        .await?
+        .iter()
+        .copied()
+        .map(|asset| async move {
+            let asset = asset.resolve().await?;
+            let path = asset.ident().path();
+            let span = tracing::info_span!("emit asset", name = path.to_string().await?.as_str());
```

Comment:
Nit:

```suggestion
            let span = tracing::info_span!("emit asset", name = %path.to_string().await?);
```

> The `%` sigil operates similarly, but indicates that the value should be recorded using its [fmt::Display](https://doc.rust-lang.org/nightly/core/fmt/trait.Display.html) implementation

https://docs.rs/tracing/latest/tracing/#recording-fields